price-estimator/README.md

```diff
@@ -2,7 +2,13 @@
 
 The api is documented with [OpenAPI](https://www.openapis.org/) in `openapi.yml`. It can be accessed in a rendered and interactive fashion on https://editor.swagger.io/ through `File -> Import file`.
 
-TODO: Add a binary or docker image that can host the interactive ui without an external service.
+Alternatively locally with docker. Note that the bind mount src path has to be absolute so we use `$PWD`.
+
+```
+docker run -p 80:8080 -e SWAGGER_JSON=/openapi.yml --mount type=bind,src="$PWD/openapi.yml",dst=/openapi.yml swaggerapi/swagger-ui
```

Comment:
When inspecting the dockerfile, I noticed: `SWAGGER_JSON_URL`. Any change this would work with a github URL?

Reply:
Indeed!

https://github.com/swagger-api/swagger-ui/blob/d9f5691f657dc64842b892255fe4a67cb6246f1b/docker/run.sh#L33-L36

Reply:
The reason I want to use the local is to make it easier to edit it. You change it and refresh the page and it will be updated.

Reply:
Makes sense! Just a cool feature that we can use if we want to build a Docker image to host the API doc that we can deploy to k8s. I think for development purposes running it with `SWAGGER_JSON` makes more sense.